libs/blocks/table/table.js

```diff
@@ -575,22 +595,44 @@ function applyStylesBasedOnScreenSize(table, originTable) {
     }
   };
 
+  const removeClones = () => {
+    const rows = table.querySelectorAll('.row');
+
+    rows.forEach((row) => {
+      const clonedCols = row.querySelectorAll('.col[data-cloned]');
+      clonedCols.forEach((col) => col.remove());
+    });
+  };
+
   // For Mobile (else: tablet / desktop)
   if (!isMerch && !table.querySelector('.row-heading .col-2')) {
     table.querySelector('.row-heading').style.display = 'block';
     table.querySelector('.row-heading .col-1').style.display = 'flex';
   }
 
   if (deviceBySize === 'MOBILE' || (isMerch && deviceBySize === 'TABLET')) {
+    removeClones();
     mobileRenderer();
   } else {
-    table.innerHTML = originTable.innerHTML;
-    reAssignEvents(table);
+    removeClones();
```

Comment:
You could also put this on L612, then you don't need to redo the action if it doesn't need to be in the if/else clause but always happens